constitution.md

```diff
@@ -2,113 +2,157 @@
 
-Have the responsibility to represent the interests of TDWG members on the Executive Committee; and to represent TDWG in appropriate meetings and other activities.
+### 6.6 Elected Representatives
 
-## Article 7. Assets
+Have the responsibility to represent the interests of TDWG members on the Executive Committee; and to represent TDWG in appropriate meetings and other activities. The number of available seats for Representatives in the Executive Committee is six (as per Article 6.7).
 
-TThe revenue of TDWG shall consist of membership dues, sales of publications, lease of rights, grants, gifts and bequests, income from investments, and any other kinds of income compatible with the aims of TDWG.
+### 6.7 Geographical distribution of Representatives
+
+The Executive Committee shall encourage participation from the membership across geographical boundaries, gender, age, ethnicity, and professional expertise. To encourage balanced representation one representative will be elected from each of the following regions: Africa, Asia, Europe, Latin America, North America, and Oceania. 
+
+## Article 7. Functional Subcommittees
+
+### 7.1 Definition, role and scope 
+
+Functional Subcommittees (FS) are committees operating under specific mandates, set by the Executive Committee in response to identified needs of TDWG. The overarching role of a FS is to assist the Executive Committee in its standing tasks. FS operate under charters prepared and approved by the Executive Committee, following consultation with the membership.   
+
+### 7.2. Establishment, duration and dissolution
+
+The scope and mode of operations for each Functional Subcommittee are defined by their published charters. Following approval by the Executive as in Paragraph 7.1, a FS is established and fully effective at the moment of election of its chairperson. FS charters shall be presented to the membership for open consultation for at least 21 days prior to approval by the Executive. Following the Executive Committee’s decision to dissolve a FS, it remains effective until the end of the FS chairperson’s term of duty, as defined in Paragraph 7.3.
+
+### 7.3 Functional subcommittee membership and leadership
+
+The chair of each FS is elected during the TDWG general elections. FS chairs are elected for two full calendar years. FS chairs are responsible for appointing a deputy-chair with attending and no voting rights at Executive Committee meetings. FS chairs invite members and manage membership in their respective committees. Each committee shall operate with more than three and fewer than 15 members. In case of resignation of a FS chair, the deputy chair will be invited to take the place of the chair until the next general elections and a new deputy is appointed.
```

Comment:
The key premise here is that Executive Committee memebrs (core officers, representatives and FS chairs) are all directly elected members. According to this, only directly elected by the membership individuals can have voting rights in the TDWG Executive Committee. Art 7 p. 3 describes the processess where a FS chair resigns. In this case the (unelected) FS deputy chair will assume as chair of the FS. Their status in the Executive Committee as non-voting members, however, will stand. The new acting-chair will also need to appoint a new deputy chair for that FS.